diode-react/src/main/scala/diode/react/ReactPot.scala

```diff
@@ -13,70 +12,70 @@ object ReactPot {
     /**
       * Render non-empty (ready or stale) content
       *
-      * @param f Transforms Pot value into a ReactNode
+      * @param f Transforms Pot value into a VdomNode
       * @return
       */
-    def render(f: A => ReactNode): ReactNode =
-      if (pot.nonEmpty) f(pot.get) else null
+    def render(f: A => VdomNode): VdomNode =
+      if (pot.nonEmpty) f(pot.get) else VdomArray.empty()
 
     /**
       * Render content in Ready state, not including stale states
       *
-      * @param f Transforms Pot value into a ReactNode
+      * @param f Transforms Pot value into a VdomNode
       * @return
       */
-    def renderReady(f: A => ReactNode): ReactNode =
-      if (pot.isReady) f(pot.get) else null
+    def renderReady(f: A => VdomNode): VdomNode =
+      if (pot.isReady) f(pot.get) else VdomArray.empty()
```

Comment:
There is probably a better way to do this than VdomArray.empty(), but I wasn't able to find an EmptyVdom equivalent that was compatible with VdomNode.